node.js

```diff
@@ -419,8 +429,10 @@ module.exports = {
     var halfYearAgo = Date.now() - TIME_TO_UPDATE_CANIUSE
 
     if (latest !== 0 && latest < halfYearAgo) {
+      var monthPassed = getMonthsPassed(latest)
+
       console.warn(
-        'Browserslist: caniuse-lite is outdated. Please run:\n' +
+        'Browserslist: caniuse-lite is '+ monthPassed +' month old. Please run:\n' +
```

Comment:
We need plural form `month`/`months` and tests

Reply:
ok, let me fix it

Reply:
fixed